docs/terminal/sydney.md

```diff
@@ -53,6 +53,26 @@ SRI is responsible for the provision of FIS in Class G airspace within the SY TM
     During a busy event, Sydney Departures is experiencing a high workload and wishes to delegate the SRI role to another TMA controller who isn't as busy.  SAN's sector is quiet, so they elect to perform the role.
 
 ## Arrival Procedures
+### STAR and Runway Assignment
+Enroute sequencing of arrivals into YSSY is a joint responsibility of ARL and BIK.
```

Comment:
Perhaps needs a rewording. ARL and BIK should not be expected to work together, ultimately it's up to SY TCU if they want any changes. So maybe something like
"Sequencing of arrivals in to YSSY is a joint responsibility between SY TCU and adjacent enroute sectors"

Reply:
Reworded to reflect what we discussed last night